src/common/metrics/weekly_new_cases_per_100k.tsx

```diff
@@ -96,16 +96,16 @@ function renderStatus(projections: Projections): React.ReactElement {
     );
   }
 
-  const newCasesPerDay = currentDailyAverageCases;
+  const newCasesPerWeek = currentDailyAverageCases * 7;
   // Try not to round cases/day to zero (since it will probably be >0 per 100k).
```

Comment:
Since this is now per week, I think we could safely remove this <1 logic and just always use `formatInteger()`